Templates/Ros2FleetRobotTemplate/Template/Gem/Include/${Name}/${Name}Bus.h

```diff
@@ -0,0 +1,40 @@
+// {BEGIN_LICENSE}
```

Comment:
question: Does this file definitely need to be added? Just checking it's not here by mistake

Reply:
That's a good question. This template may be used in a variety of different robotic projects. I'm not sure what's the minimum necessary set of files in such a case, so we took the O3DE [DefaultProject template](https://github.com/o3de/o3de/tree/development/Templates/DefaultProject) as a basis for it. 
This file is also present in the [MinimalProject template](https://github.com/o3de/o3de/tree/development/Templates/MinimalProject).